sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BridgeInternal.java

```diff
@@ -199,6 +199,15 @@ public static void setQueryPlanDiagnosticsContext(CosmosDiagnostics cosmosDiagno
         cosmosDiagnostics.getFeedResponseDiagnostics().setDiagnosticsContext(diagnosticsContext);
     }
 
+    @Warning(value = INTERNAL_USE_ONLY_WARNING)
+    public static QueryInfo.QueryPlanDiagnosticsContext getQueryPlanDiagnosticsContext(CosmosDiagnostics cosmosDiagnostics) {
```

Comment:
I thought we agreed to not add more BridgeInternal methods. This will trigger review discussed with the central SDK team.

can we instead use the same technique used here: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ImplementationBridgeHelpers.java

Reply:
@kushagraThapar  thoughts ?, are we stop using for Cosmos also, i thought that is for supporting project. This approach comes with cost of multiple code lines.

Reply:
Done moved to ImplementationBridgeHelper

Reply:
thanks awesome. Thanks @simplynaveen20